docs/0.3-DRAFT/index.md

```diff
@@ -1586,7 +1620,9 @@ For example, this data is exported from an [Omeka](https://omeka.org) repository
 }
 ```
 
+[See an example of a repository exported to a RO-Crate.](https://data.research.uts.edu.au/examples/v1.0/farms_to_freeways/)
```

Comment:
Old text? I thought you agreed we could not link to these yet without stable URIs? Example is also outdated datacrate 1.0 rather than RO-Crate.